collect_app/src/main/java/org/odk/collect/android/widgets/BaseImageWidget.java

```diff
@@ -313,4 +314,32 @@ protected void launchActivityForResult(Intent intent, final int resourceCode, fi
             cancelWaitingForData();
         }
     }
+
+    private File addDefaultImageIfExists(File f) {
+        File defaultFile = new File(getDefaultFilePath());
+        if (defaultFile.exists()) {
+            binaryName = System.currentTimeMillis() + "." + FilenameUtils.getExtension(defaultFile.getName());
+            FileUtils.copyFile(defaultFile, new File(getInstanceFolder() + File.separator + binaryName));
+            f = new File(getInstanceFolder() + File.separator + binaryName);
+        }
+        return f;
+    }
+
+    private String getDefaultFilePath() {
+        String defaultFilePath = null;
+        FormController formController = Collect.getInstance().getFormController();
+        if (formController != null) {
+            if (binaryName.startsWith("jr://images/")) {
+                try {
+                    defaultFilePath = referenceManager.deriveReference(binaryName).getLocalURI();
+                } catch (InvalidReferenceException e) {
+                    Timber.w(e);
+                }
+            } else {
+                defaultFilePath = formController.getMediaFolder() + File.separator + binaryName;
```

Comment:
This means you're also trying to manually get things from the media folder if a file has no prefix? Why? Also consider that if a `jr://file/` URI is set, for example, you'd end up with a path that looks like `/path/to/media/folder/jr://file/name`

Reply:
Ok I assumed that we should support both with and without prefix. I'll get rid of it.